addons/nuxeo-template-rendering/nuxeo-template-rendering-core/src/main/java/org/nuxeo/template/adapters/doc/TemplateBinding.java

```diff
@@ -18,14 +18,16 @@
  */
 package org.nuxeo.template.adapters.doc;
 
+import org.nuxeo.runtime.api.Framework;
+import org.nuxeo.template.api.TemplateInput;
+import org.nuxeo.template.serializer.executors.Serializer;
+import org.nuxeo.template.serializer.service.SerializerService;
+
 import java.io.Serializable;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
-import org.nuxeo.template.XMLSerializer;
```

Comment:
In our formatting style `org` is after `java`, like it was originally in the file.

Reply:
I embedded this fix in a commit of the history. But I dit a [specific commit for other classes that needed an reordered imports](https://github.com/clockworx-com/nuxeo/commit/957bbefb671863d5c2052f9542526a73d18616c9).